docs/config.md

```diff
@@ -0,0 +1,71 @@
+# Integrate WSO2 API Manager 3.2.0 with Auth0
```

Comment:
do we need the version?

Reply:
title doesn't need the version. we can include that in the pre requisits

Reply:
This has changed and readme updated.